nishant_work/while_for.py

```diff
@@ -0,0 +1,33 @@
+#What is different of while and for loop? which one is fast & why ? Write program for example.
+
```

Comment:
All answer is correct, very good! but you need to change the program over here, you need to show with bigger loop which on is faster using timestamp